hastings-center-report.csl

```diff
@@ -0,0 +1,1259 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style class="note" version="1.0" et-al-min="4" et-al-use-first="1" initialize-with=". " page-range-format="chicago" demote-non-dropping-particle="never" xmlns="http://purl.org/net/xbiblio/csl">
+<!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>Hastings Center Report</title>
+    <title-short>HCR</title-short>
+    <id>http://www.zotero.org/styles/hastings-center-report</id>
+    <link href="http://www.zotero.org/styles/hastings-center-report" rel="self"/>
+    <link href="http://www.zotero.org/styles/chicago-fullnote-bibliography" rel="template"/>
+    <link href="http://www.thehastingscenter.org/Publications/HCR/Guidelines.aspx" rel="documentation"/>
+    <author>
+      <name>Warren Whipple</name>
+      <email>modalrealist@gmail.com</email>
+    </author>
+    <contributor>
+      <name>Julian Onions</name>
+      <email>julian.onions@gmail.com</email>
+    </contributor>
+    <contributor>
+      <name>Simon Kornblith</name>
+      <email>simon@simonster.com</email>
+    </contributor>
+    <contributor>
+      <name>Elena Razlogova</name>
+      <email>elena.razlogova@gmail.com</email>
+    </contributor>
+    <contributor>
+      <name>Frank Bennett</name>
+      <email>biercenator@gmail.com</email>
+    </contributor>
+    <contributor>
+      <name>Andrew Dunning</name>
+      <email>andrew.dunning@utoronto.ca</email>
+    </contributor>
```

Comment:
Please remove all contributors from chicago-fullnote-bibliography.csl. The template link is enough attribution.